plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/struct/JDBCTableColumn.java

```diff
@@ -200,6 +200,15 @@ public List<DBDLabelValuePair> getValueEnumeration(
         query.append("\nFROM ").append(DBUtils.getObjectFullName(getTable(), DBPEvaluationContext.DML));
         if (valuePattern instanceof String) {
             query.append("\nWHERE ");
+            boolean needCast = false;
+            String typeExt = "";
+            DBSDataType dataType = DBUtils.getDataType(this);
+            if (dataType != null) typeExt = CommonUtils.toString(dataType.geTypeExtension());
```

Comment:
@E1izabeth , nota bene - possibly there is typo in existing code geTypeExtension ("ge", not "get") - but I left it unchanged, I dont know all the cases and dependencies

Reply:
Oh, I see. We will fix it later out of scope of this PR.